certs/client.key

```diff
@@ -0,0 +1,16 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Yes, indeed.  In fact I was thinking of going further and deleting them, and instead adding the instructions that we need to the test procedure to regenerate them.  WDYT?